components/Payment/Payment.tsx

```diff
@@ -54,50 +54,70 @@ function Payment(props: Props) {
+                                        variant="success"
+                                        onClick={() => {
+                                            onPayPaypal(payPalProductId)
+                                        }}
+                                        className={styles.paymentButton}
+                                        disabled={!props.cancellationRightLossConfirmed}
+                                    >
+                                        {payPalProductId === isLoadingId ? (
+                                            <p className={styles.manualRedirectLink}>
+                                                Redirecting to PayPal...
+                                                <br /> Not working?{' '}
+                                                <a href={currentRedirectLink} target="_blank">
+                                                    Click here
+                                                </a>
+                                            </p>
+                                        ) : (
+                                            `${numberWithThousandsSeperators(Math.round(paypalPrice * 100) / 100)} Euro`
+                                        )}
+                                    </Button>
+                                </div>
+                            }
+                        />
                     </p>
                     <p className={styles.paymentOption}>
-                        <div style={{ width: '50%' }}>Buy with Stripe</div>
-                        <Button
-                            variant="success"
-                            onClick={() => {
-                                onPayStripe(stripeProductId)
-                            }}
-                            style={{ width: '40%' }}
-                            disabled={props.disabled}
-                        >
-                            {stripeProductId === isLoadingId ? (
-                                <p className={styles.manualRedirectLink}>
-                                    Redirecting to Stripe...
-                                    <br />
-                                    Not working?{' '}
-                                    <a href={currentRedirectLink} target="_blank">
-                                        Click here
-                                    </a>
-                                </p>
-                            ) : (
-                                `${numberWithThousandsSeperators(Math.round(stripePrice * 100) / 100)} Euro`
-                            )}
-                        </Button>
+                        <div className={styles.paymentLabel}>Buy with Stripe</div>
+                        <Tooltip
+                            type="hover"
+                            tooltipContent={
+                                !props.cancellationRightLossConfirmed ? <span>Please note the information regarding your cancellation right above.</span> : null
```

Comment:
the message content (and check) is duplicated, please refactor it so it only exists once